### PR TITLE
Add flask-rest-jsonapi 0.12.2 to conda-forge.

### PR DESCRIPTION
**Pull request by sodre**
This PR includes the dependencies for that project as well.

### CODE REVIEW

**Comment by sodre** [comment]
@conda-forge/core flask-rest-jsonapi is ready for review.

File: recipes/flask-rest-jsonapi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - pytest-runner
+  run:
+    - python
+    - six
+    - flask >=0.11
+    - marshmallow-jsonapi
+    - sqlalchemy
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
```
Comment (synapticarbors) [inline recipes/flask-rest-jsonapi/meta.yaml]: Please remove all comments.

File: recipes/flask-rest-jsonapi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Flask-REST-JSONAPI is a flask extension for building REST APIs'
+
+  description: |
+      Flask-REST-JSONAPI is a flask extension for building REST APIs. It combines the power of
+      Flask-Restless and the flexibility of Flask-RESTful around a strong specification
+      JSONAPI 1.0. This framework is designed to quickly build REST APIs and fit the
+      complexity of real life projects with legacy data and multiple data storages
+  doc_url: http://{{ name }}.readthedocs.io/
```
Comment (synapticarbors) [inline recipes/flask-rest-jsonapi/meta.yaml]: Please don't use jinja template variables in non-source urls.

File: recipes/flask-rest-jsonapi/meta.yaml (hunk 1 above)
Comment (sodre) [inline recipes/flask-rest-jsonapi/meta.yaml]: no problem.  It seems I always miss one of them

**Pull request merged**